zadanie2/src/Zadanie2.java

```diff
@@ -0,0 +1,62 @@
+import java.text.SimpleDateFormat;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.util.Scanner;
+
+public class Zadanie2 {
+    public static void main(String[] args) {
+        Data data = new Data();
+        data.sprawdzDate();
+    }
+}
+
+/**
+* Program sprawdza poprawnosc daty podanej przez uzytkownika.
+ * Daty spoza przedzialu <2001-2099> sa uznawane za niepoprawne.
+ *
+ * @author Kamil Best
+ */
+class Data {
+    final String DATE_FORMAT = "dd-MM-yyyy";
+    private String data;
+    private String rok;
+    private int rokLiczba;
+
+    private void pobierzDate() {
```

Comment:
to mogloby byc publiczne, kolejna metoda moglaby sprawdzic, czy cos pobrane zostalo inaczej pluc wyjatkiem